seerep-hdf5/seerep-hdf5-core/include/seerep-hdf5-core/hdf5-core-general.h

```diff
@@ -89,6 +90,12 @@ class Hdf5CoreGeneral
   void writeProjectFrameId(const std::string& frameId);
   std::string readProjectFrameId();
 
+  //################
+  // Geodetic Coordinates
+  //################
+  void writeGeodeticLocation(const seerep_core_msgs::GeodeticCoordinates geocoords);
+  std::optional<seerep_core_msgs::GeodeticCoordinates> readGeodeticLocation();
```

Comment:
Eventhough, this file has no code docu yet, please write documentation for new functions